functional_offset_manager_test.go

```diff
@@ -24,10 +19,6 @@ func TestFuncOffsetManager(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	if _, err := offsetManager.ManagePartition("does_not_exist", 123); err != ErrUnknownTopicOrPartition {
-		t.Fatal("Expected ErrUnknownTopicOrPartition when starting a partition offset manager for a partition that does not exist, got:", err)
-	}
-
```

Comment:
Why did this have to be removed? Is it because the new group protocol is abstract, and potentially could work for other sources besides kafka partitions? (e.g. Kafka Connect). Or is there another reason?